### PR TITLE
Format accept language code

### PR DESCRIPTION
**Pull request by prodis**
Currently:
```ruby
language = Browser::AcceptLanguage.new('EN-GB')

language.code 
# => "EN"

language.full
# => "EN-GB"
```

Now:
```ruby
language = Browser::AcceptLanguage.new('EN-GB')

language.code 
# => "en"

language.full
# => "en-GB"
```

### CODE REVIEW

**Comment by fnando** [comment]
Hello, @prodis! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/fnando/browser/pulls/285). Please check our [documentation](https://ebertapp.io/docs) for more information.

**Comment by prodis** [comment]
@fnando

File: test/unit/accept_language_test.rb [hunk 1]
```diff
@@ -4,15 +4,23 @@
...
     assert_equal expect[:quality], item.quality, "failed quality comparison"
   end
 
+  # full
   test "returns full language" do
     language = Browser::AcceptLanguage.new("en-GB")
     assert_equal "en-GB", language.full
   end
 
+  # name
```
Comment (fnando) [inline test/unit/accept_language_test.rb]: Drop this comment.

File: test/unit/accept_language_test.rb [hunk 2]
```diff
@@ -4,15 +4,23 @@
...
+    if expect[:region].nil?
+      assert_nil item.region, "failed region comparison"
+    else
+      assert_equal expect[:region], item.region, "failed region comparison"
+    end
+
     assert_equal expect[:quality], item.quality, "failed quality comparison"
   end
 
+  # full
```
Comment (fnando) [inline test/unit/accept_language_test.rb]: Drop this comment.

File: test/unit/accept_language_test.rb [hunk 3]
```diff
@@ -26,6 +34,38 @@ def assert_language(item, expect = {})
     assert_nil language.name
   end
 
+  # code
```
Comment (fnando) [inline test/unit/accept_language_test.rb]: Drop this comment.

File: test/unit/accept_language_test.rb [hunk 4]
```diff
@@ -26,6 +34,38 @@ def assert_language(item, expect = {})
...
+  end
+
+  test "returns formatted code" do
+    %w[EN-GB En-GB eN-GB].each do |locale|
+      language = Browser::AcceptLanguage.new(locale)
+      assert_equal "en", language.code
+    end
+  end
+
+  # region
```
Comment (fnando) [inline test/unit/accept_language_test.rb]: Drop this comment.

File: test/unit/accept_language_test.rb [hunk 5]
```diff
@@ -26,6 +34,38 @@ def assert_language(item, expect = {})
...
+      assert_equal "GB", language.region
+    end
+  end
+
+  test "returns nil for language without region" do
+    language = Browser::AcceptLanguage.new("en")
+    assert_nil language.region
+  end
+
+  # new
```
Comment (fnando) [inline test/unit/accept_language_test.rb]: Drop this comment.

File: test/unit/accept_language_test.rb [hunk 6]
```diff
@@ -46,6 +86,7 @@ def assert_language(item, expect = {})
     assert_language language, code: "az", region: "AZ", quality: 1.0
   end
 
+  # parse
```
Comment (fnando) [inline test/unit/accept_language_test.rb]: Drop this comment.

File: test/unit/accept_language_test.rb [hunk 7]
```diff
@@ -4,15 +4,23 @@
 class AcceptLanguageTest < Minitest::Test
   def assert_language(item, expect = {})
     assert_equal expect[:code], item.code, "failed code comparison"
-    assert_equal expect[:region], item.region, "failed region comparison"
+
+    if expect[:region].nil?
```
Comment (fnando) [inline test/unit/accept_language_test.rb]: Why is this being added if you're not calling it anywhere on tests you wrote in this PR?

Comment (fnando) [inline test/unit/accept_language_test.rb]: Minitest deprecations. Now I get it.

**Comment by fnando** [comment]
Close this in favor of #298.

**Pull request closed without merging**